src/models/childModel.js

```diff
@@ -0,0 +1,39 @@
+const mongoose = require('mongoose');
+
+const childSchema = new mongoose.Schema({
+    name: {
+        type: String,
+        required: true
+    },
+    tasks: [
+        {
+            taskId: {
+                type: mongoose.Schema.ObjectId,
+                ref: 'Tasks',
+            },
+            isCompleted: {
+                type: Boolean,
+            }
+        }
+    ],
+    rewards: [
```

Comment:
so what you need to capture is the rewardId, this can then be referenced in the queries to get other reward info such as `receivedAt`